azure/template.json

```diff
@@ -14,10 +14,10 @@
         "serviceName": {
             "type": "string"
         },
-        "sharedFrontEndAppServicePlan": {
+        "SharedBackEndAppServicePlanName": {
```

Comment:
Change to camelCase to match other parameters 👍

Reply:
Fixed